pandas/io/tests/test_pytables.py

```diff
@@ -927,6 +928,28 @@ def test_encoding(self):
             result = store.select('df',Term('columns=A',encoding='ascii'))
             tm.assert_frame_equal(result,expected)
 
+    def test_latin_encoding(self):
+        examples = [
+            pandas.Series(['EÉ, 17', '', 'a', 'b', 'c'], dtype='category'),
+            pandas.Series(['EÉ, 17', 'a', 'b', 'c'], dtype='category'),
+            pandas.Series(['', 'a', 'b', 'c'], dtype='category'),
+            pandas.Series(['EE, 17', '', 'a', 'b', 'c'], dtype='category'),
+            pandas.Series(['øü', 'a', 'b', 'c'], dtype='category'),
+            pandas.Series(['Aøü', '', 'a', 'b', 'c'], dtype='category'),
+            ]
+
+        def roundtrip(s, key='data'):
```

Comment:
use ``ensure_clean_path`` (or simpler to use use ``ensure_clean_store`` if you remove the existing key each time)